packages/frontend/src/components/common/modal/ChartCreateModal/SaveToDashboard.tsx

```diff
@@ -158,7 +158,7 @@ export const SaveToDashboard: FC<Props> = ({
             <Stack p="md">
                 <Stack spacing="xs">
                     <TextInput
-                        label="Enter a memorable name for your chart"
+                        label="Chart Name"
```

Comment:
@PriPatel do we want `Name` uppercase? because we have `Chart description` bellow it.

Reply:
Hey.
I made the required changes. If any further changes required, would be happy to make them :smile: